storage/ci/phinx.php.example

```diff
@@ -11,7 +11,7 @@ return [
         'production'              => [
             'adapter' => 'mysql',
             'host'    => getenv('DATA_API_MYSQL_HOST'),
-            'name'    => 'gonano',
+            'name'    => getenv('DATA_API_MYSQL_NAME'),
```

Comment:
Shouldn't we use `Niden\Core\envValue` here?

Reply:
Probably will be change to `getenv` as is now. I'll see if `envValue()` function is needed